experiments/standalone/Soil/water_conservation.jl

```diff
@@ -123,14 +129,14 @@ for FT in (Float32, Float64)
 
         if FT == Float64
             # Calculate water mass balance over entire simulation
-            mass_end = sum(sol.u[end].soil.ϑ_l)
-            mass_start = sum(sol.u[1].soil.ϑ_l)
-            t_sim = sol.t[end] - sol.t[1]
-            # Flux changes water content every timestep (assumes constant flux_in, flux_out)
-            mass_change_exp = -(flux_in - flux_out) * t_sim
-            mass_change_actual = mass_end - mass_start
+            mass_end = sv.saveval[end].soil.total_water
+            mass_start = sv.saveval[1].soil.total_water
+            ∫Fdt_end = sol.u[end].soil.∫Fwdt
+            ∫Fdt_start = sol.u[1].soil.∫Fwdt
+            mass_change_exp = parent(∫Fdt_end .- ∫Fdt_start)[1]
+            mass_change_actual = parent(mass_end .- mass_start)[1]
```

Comment:
If we are justing just the first and the last time, we don't need the SavingCallback: we can save the state at the beginning, and compare it to the one at the end